lib/cli.rb

```diff
@@ -571,9 +571,18 @@ def self.parse_scopes(scope_string)
 
       filter = FilterOptionParser.parse("show", options, global_options)
 
-      if options["verbose"] && !filter.empty?
-        Machinery::Ui.puts "\nThe following filters were applied before showing the description:"
-        Machinery::Ui.puts filter.to_array.join("\n") + "\n\n"
+      inspected_filters = description.filter_definitions("inspect")
+
+      if options["verbose"]
+        if !inspected_filters.empty?
+          Machinery::Ui.puts "\nThe following filters were applied during inspectation:"
+          Machinery::Ui.puts inspected_filters.join("\n") + "\n\n"
+        end
+
+        if !filter.empty?
+          Machinery::Ui.puts "\nThe following filters were applied before showing the description:"
+          Machinery::Ui.puts filter.to_array.join("\n") + "\n\n"
+        end
```

Comment:
Yes, but I had to add another if-statement. There is only an output, if the inspected_filter or/and the filter array are not empty.